processor/src/main/java/org/mapstruct/ap/spi/DefaultAccessorNamingStrategy.java

```diff
@@ -231,4 +231,8 @@ public String getCollectionGetterName(String property) {
             + "future versions." );
     }
 
+    @Override
+    public String toString() {
+        return getClass().getCanonicalName();
```

Comment:
Why do we need to use `getClass().getCanonicalName()`?  We could do that on the caller side, not sure if it is a good `toString` method